templates/Gemfile.erb

```diff
@@ -2,53 +2,104 @@ source "https://rubygems.org"
 
 ruby "<%= Suspenders::RUBY_VERSION %>"
 
-gem "airbrake"
-gem "bourbon", "~> 4.1.0"
-gem "coffee-rails", "~> 4.1.0"
-gem "delayed_job_active_record"
-gem "email_validator"
-gem "flutie"
-gem "high_voltage"
-gem "i18n-tasks"
-gem "jquery-rails"
-gem "neat", "~> 1.7.0"
-gem "newrelic_rpm"
-gem "normalize-rails", "~> 3.0.0"
+# gem "airbrake"
```

Comment:
hmmm I don't love generated files with tons of comments. I would personally rather see gems that are required and then nothing else.